examples/agents/e2e_loop_with_custom_tools.py

```diff
@@ -28,7 +28,7 @@ async def run_main(host: str, port: int, disable_safety: bool = False):
         print(f"Available shields found: {available_shields}")
 
     available_models = [model.identifier for model in client.models.list()]
-    supported_models = [x for x in available_models if "3.2" in x]
+    supported_models = [x for x in available_models if "3.2" in x and "Vision" not in x]
```

Comment:
llama3.1 and llama3.2 multimodal models follow the same tool prompt format, and don't have ToolPromptFormat ToolPromptFormat.python_list